lib/DOMutil.js

```diff
@@ -135,8 +135,8 @@ exports.getDOMElement = function (elementType, JSONcontainer, DOMContainer, inse
  * The reason the JSONcontainer and the target SVG svgContainer have to be supplied is so the legend can use these functions
  * as well.
  *
- * @param {Number} x
- * @param {Number} y
+ * @param {number} x
+ * @param {number} y
```

Comment:
TIL to use lower-case for `string` and `number` in jsdoc. Will cherish this information.